2.0.0-alpha.2

matteobruni opened the pull request.

matteobruni committed:
build: version bump

matteobruni committed:
build: fixed imports in editor

ghost commented:
## DeepCode's analysis on [#29920a](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/e471850859c6d73cfd7233765f0adb88c9ed5a3c/matteobruni/tsparticles/29920a6b994c4698e559827b3846b82c37ced5f2/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) found:

- :information_source: **1** minor issue. :point_down:
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">'public' is implicit. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/29920a6b994c4698e559827b3846b82c37ced5f2/editor/src/Sections/Options/Infection/InfectionOptionsEditor.ts#L8">InfectionOptionsEditor.ts:8</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/29920a6b994c4698e559827b3846b82c37ced5f2/editor/src/Sections/Options/Infection/InfectionOptionsEditor.ts#L15">InfectionOptionsEditor.ts:15</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/e471850859c6d73cfd7233765f0adb88c9ed5a3c/matteobruni/tsparticles/29920a6b994c4698e559827b3846b82c37ced5f2/pr/_/%2Feditor%2Fsrc%2FSections%2FOptions%2FInfection%2FInfectionOptionsEditor.ts/javascript%2Ftslint%2Fmember-access/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/e471850859c6d73cfd7233765f0adb88c9ed5a3c/matteobruni/tsparticles/29920a6b994c4698e559827b3846b82c37ced5f2/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

dependabot[bot] committed:
build(deps-dev): bump eslint-plugin-vue from 7.4.0 to 7.5.0

Bumps [eslint-plugin-vue](https://github.com/vuejs/eslint-plugin-vue) from 7.4.0 to 7.5.0.
- [Release notes](https://github.com/vuejs/eslint-plugin-vue/releases)
- [Commits](https://github.com/vuejs/eslint-plugin-vue/compare/v7.4.0...v7.5.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump svelte-check from 1.1.24 to 1.1.28

Bumps [svelte-check](https://github.com/sveltejs/language-tools) from 1.1.24 to 1.1.28.
- [Release notes](https://github.com/sveltejs/language-tools/releases)
- [Commits](https://github.com/sveltejs/language-tools/commits)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1235 from matteobruni/dependabot/npm_and_yarn/dev/eslint-plugin-vue-7.5.0

build(deps-dev): bump eslint-plugin-vue from 7.4.0 to 7.5.0

matteobruni committed:
Merge pull request #1236 from matteobruni/dependabot/npm_and_yarn/dev/svelte-check-1.1.28

build(deps-dev): bump svelte-check from 1.1.24 to 1.1.28

dependabot[bot] committed:
build(deps-dev): bump eslint-plugin-prettier from 3.3.0 to 3.3.1

Bumps [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier) from 3.3.0 to 3.3.1.
- [Release notes](https://github.com/prettier/eslint-plugin-prettier/releases)
- [Changelog](https://github.com/prettier/eslint-plugin-prettier/blob/master/CHANGELOG.md)
- [Commits](https://github.com/prettier/eslint-plugin-prettier/compare/v3.3.0...v3.3.1)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump svelte-preprocess from 4.6.1 to 4.6.3

Bumps [svelte-preprocess](https://github.com/sveltejs/svelte-preprocess) from 4.6.1 to 4.6.3.
- [Release notes](https://github.com/sveltejs/svelte-preprocess/releases)
- [Changelog](https://github.com/sveltejs/svelte-preprocess/blob/main/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/svelte-preprocess/compare/v4.6.1...v4.6.3)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
chore(react): replaced loadsh/isEqual with fast-deep-equal

matteobruni committed:
Merge remote-tracking branch 'origin/dev' into dev

matteobruni committed:
Merge pull request #1239 from matteobruni/dependabot/npm_and_yarn/dev/svelte-preprocess-4.6.3

build(deps-dev): bump svelte-preprocess from 4.6.1 to 4.6.3

matteobruni committed:
Merge pull request #1237 from matteobruni/dependabot/npm_and_yarn/dev/eslint-plugin-prettier-3.3.1

build(deps-dev): bump eslint-plugin-prettier from 3.3.0 to 3.3.1

dependabot[bot] committed:
build(deps-dev): bump postcss from 8.2.2 to 8.2.4

Bumps [postcss](https://github.com/postcss/postcss) from 8.2.2 to 8.2.4.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.2.2...8.2.4)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump typescript-json-schema from 0.47.0 to 0.48.0

Bumps [typescript-json-schema](https://github.com/YousefED/typescript-json-schema) from 0.47.0 to 0.48.0.
- [Release notes](https://github.com/YousefED/typescript-json-schema/releases)
- [Commits](https://github.com/YousefED/typescript-json-schema/compare/v0.47.0...v0.48.0)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
chore(components): replaced loadsh/isEqual with fast-deep-equal in all components

matteobruni committed:
chore(components): replaced loadsh/isEqual with fast-deep-equal in all components

dependabot[bot] committed:
build(deps): bump preact from 10.5.8 to 10.5.11

Bumps [preact](https://github.com/preactjs/preact) from 10.5.8 to 10.5.11.
- [Release notes](https://github.com/preactjs/preact/releases)
- [Commits](https://github.com/preactjs/preact/compare/10.5.8...10.5.11)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
chore(components): added the loaded property on react, preact and inferno components, closes #1218

matteobruni committed:
Merge pull request #1241 from matteobruni/dependabot/npm_and_yarn/dev/postcss-8.2.4

build(deps-dev): bump postcss from 8.2.2 to 8.2.4

matteobruni committed:
Merge pull request #1242 from matteobruni/dependabot/npm_and_yarn/dev/typescript-json-schema-0.48.0

build(deps-dev): bump typescript-json-schema from 0.47.0 to 0.48.0

dependabot[bot] committed:
build(deps-dev): bump uglify-js from 3.12.3 to 3.12.5

Bumps [uglify-js](https://github.com/mishoo/UglifyJS) from 3.12.3 to 3.12.5.
- [Release notes](https://github.com/mishoo/UglifyJS/releases)
- [Commits](https://github.com/mishoo/UglifyJS/compare/v3.12.3...v3.12.5)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1243 from matteobruni/dependabot/npm_and_yarn/dev/preact-10.5.11

build(deps): bump preact from 10.5.8 to 10.5.11

dependabot[bot] committed:
build(deps-dev): bump rollup-plugin-svelte from 7.0.0 to 7.1.0

Bumps [rollup-plugin-svelte](https://github.com/sveltejs/rollup-plugin-svelte) from 7.0.0 to 7.1.0.
- [Release notes](https://github.com/sveltejs/rollup-plugin-svelte/releases)
- [Changelog](https://github.com/sveltejs/rollup-plugin-svelte/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/rollup-plugin-svelte/compare/v7.0.0...v7.1.0)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1244 from matteobruni/dependabot/npm_and_yarn/dev/uglify-js-3.12.5

build(deps-dev): bump uglify-js from 3.12.3 to 3.12.5

matteobruni committed:
Merge pull request #1245 from matteobruni/dependabot/npm_and_yarn/dev/rollup-plugin-svelte-7.1.0

build(deps-dev): bump rollup-plugin-svelte from 7.0.0 to 7.1.0

dependabot[bot] committed:
build(deps): bump svelte from 3.31.0 to 3.31.2

Bumps [svelte](https://github.com/sveltejs/svelte) from 3.31.0 to 3.31.2.
- [Release notes](https://github.com/sveltejs/svelte/releases)
- [Changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/svelte/compare/v3.31.0...v3.31.2)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @vue/cli-plugin-eslint from 4.5.9 to 4.5.11

Bumps [@vue/cli-plugin-eslint](https://github.com/vuejs/vue-cli/tree/HEAD/packages/@vue/cli-plugin-eslint) from 4.5.9 to 4.5.11.
- [Release notes](https://github.com/vuejs/vue-cli/releases)
- [Changelog](https://github.com/vuejs/vue-cli/blob/v4.5.11/CHANGELOG.md)
- [Commits](https://github.com/vuejs/vue-cli/commits/v4.5.11/packages/@vue/cli-plugin-eslint)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump ts-loader from 8.0.13 to 8.0.14

Bumps [ts-loader](https://github.com/TypeStrong/ts-loader) from 8.0.13 to 8.0.14.
- [Release notes](https://github.com/TypeStrong/ts-loader/releases)
- [Changelog](https://github.com/TypeStrong/ts-loader/blob/master/CHANGELOG.md)
- [Commits](https://github.com/TypeStrong/ts-loader/compare/v8.0.13...v8.0.14)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1246 from matteobruni/dependabot/npm_and_yarn/dev/svelte-3.31.2

build(deps): bump svelte from 3.31.0 to 3.31.2

matteobruni committed:
Merge pull request #1247 from matteobruni/dependabot/npm_and_yarn/dev/vue/cli-plugin-eslint-4.5.11

build(deps-dev): bump @vue/cli-plugin-eslint from 4.5.9 to 4.5.11

matteobruni committed:
Merge pull request #1248 from matteobruni/dependabot/npm_and_yarn/dev/ts-loader-8.0.14

build(deps-dev): bump ts-loader from 8.0.13 to 8.0.14

matteobruni committed:
build: updated angular demo

matteobruni committed:
build: updated angular component yarn.lock

matteobruni committed:
chore(release): published new version

 - ng-particles@3.0.0-alpha.2
 - inferno-particles@2.0.0-alpha.2
 - jquery-particles@2.0.0-alpha.2
 - preact-particles@2.0.0-alpha.2
 - react-tsparticles@2.0.0-alpha.2
 - svelte-particles@2.0.0-alpha.2
 - particles.vue@3.0.0-alpha.2
 - particles.vue3@2.0.0-alpha.2
 - ng-particles-demo@2.0.0-alpha.2
 - editor-demo@2.0.0-alpha.2
 - inferno-particles-demo@2.0.0-alpha.2
 - jquery-particles-demo@2.0.0-alpha.2
 - tsparticles-demo@2.0.0-alpha.2
 - preact-particles-demo@2.0.0-alpha.2
 - react-tsparticles-demo@2.0.0-alpha.2
 - svelte-demo@2.0.0-alpha.2
 - particles.vue-demo@2.0.0-alpha.2
 - particles.vue3-demo@2.0.0-alpha.2
 - tsparticles-editor@2.0.0-alpha.2
 - tsparticles-interaction-light@2.0.0-alpha.2
 - tsparticles-interaction-external-attract@2.0.0-alpha.2
 - tsparticles-interaction-external-bounce@2.0.0-alpha.2
 - tsparticles-interaction-external-bubble@2.0.0-alpha.2
 - tsparticles-interaction-external-connect@2.0.0-alpha.2
 - tsparticles-interaction-external-grab@2.0.0-alpha.2
 - tsparticles-interaction-external-repulse@2.0.0-alpha.2
 - tsparticles-interaction-external-trail@2.0.0-alpha.2
 - tsparticles-interaction-particles-attract@2.0.0-alpha.2
 - tsparticles-interaction-particles-collisions@2.0.0-alpha.2
 - tsparticles-interaction-particles-links@2.0.0-alpha.2
 - tsparticles-interaction-particles-repulse@2.0.0-alpha.2
 - tsparticles-core@2.0.0-alpha.2
 - tsparticles@2.0.0-alpha.2
 - tsparticles-slim@2.0.0-alpha.2
 - tsparticles-plugin-absorbers@2.0.0-alpha.2
 - tsparticles-plugin-emitters@2.0.0-alpha.2
 - tsparticles-plugin-infection@2.0.0-alpha.2
 - tsparticles-plugin-polygon-mask@2.0.0-alpha.2
 - tsparticles-preset-big-circles@2.0.0-alpha.2
 - tsparticles-preset-fire@2.0.0-alpha.2
 - tsparticles-preset-fountain@2.0.0-alpha.2
 - tsparticles-preset-links@2.0.0-alpha.2
 - tsparticles-preset-sea-anemone@2.0.0-alpha.2
 - tsparticles-preset-snow@2.0.0-alpha.2
 - tsparticles-preset-stars@2.0.0-alpha.2
 - tsparticles-shape-bubble@2.0.0-alpha.2
 - tsparticles-shape-circle@2.0.0-alpha.2
 - tsparticles-shape-heart@2.0.0-alpha.2
 - tsparticles-shape-image@2.0.0-alpha.2
 - tsparticles-shape-line@2.0.0-alpha.2
 - tsparticles-shape-multiline-text@2.0.0-alpha.2
 - tsparticles-shape-polygon@2.0.0-alpha.2
 - tsparticles-shape-spiral@2.0.0-alpha.2
 - tsparticles-shape-square@2.0.0-alpha.2
 - tsparticles-shape-star@2.0.0-alpha.2
 - tsparticles-shape-text@2.0.0-alpha.2
 - cra-template-particles-typescript@2.0.0-alpha.2
 - cra-template-particles@2.0.0-alpha.2
 - tsparticles-updater-angle@2.0.0-alpha.2
 - tsparticles-updater-color@2.0.0-alpha.2
 - tsparticles-updater-life@2.0.0-alpha.2
 - tsparticles-updater-opacity@2.0.0-alpha.2
 - tsparticles-updater-orbit@2.0.0-alpha.2
 - tsparticles-updater-out-modes@2.0.0-alpha.2
 - tsparticles-updater-size@2.0.0-alpha.2
 - tsparticles-updater-stroke-color@2.0.0-alpha.2

matteobruni committed:
build: version bump
